Frontend/src/components/Input/ButtonTight.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+  export let text;
+  export let onclick;
+</script>
+
+<button class="button-tight" on:click={onclick}>{text}</button>
```

Comment:
You should use an event, not a prop for onclick
```suggestion
  export let text;
</script>

<button class="button-tight" on:click>{text}</button>
```